### PR TITLE
Add Game Theory Description

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -31,7 +31,7 @@
 \IEEEauthorblockA{\IEEEauthorrefmark{1}\IEEEauthorrefmark{2}Tainan City, Taiwan (R.O.C.), \IEEEauthorrefmark{3}Taipei City, Taiwan (R.O.C.)} \\
 
 %%% Fill in email here
-\IEEEauthorblockA{\IEEEauthorrefmark{1}jserv@ccns.ncku.edu.tw, \IEEEauthorrefmark{2}jkrvivian@gmail.com, \IEEEauthorrefmark{3}twzjwang@gmail.com}
+\IEEEauthorblockA{\IEEEauthorrefmark{1}jserv@ccns.ncku.edu.tw, \IEEEauthorrefmark{2}jkrvivian@gmail.com, \IEEEauthorrefmark{3}twzjwang@gmail.com, \IEEEauthorrefmark{4}yuanyanghau@gmail.com}
```
Comment (jserv) [inline paper.tex]: You shall use BiiLabs' email address for formal writing.

File: paper.tex [hunk 2]
```diff
@@ -136,6 +136,34 @@ \subsection{Refunding}
 
 Where $price$  is the subscription price, $M$ is the number of expected data samples, $F_{b}$ is the brokerage fee(\%), $F_{t}$ is the transaction fee of the smart contract(\%), $N$ is the number of subscribers in this contract.
 
+\subsection{Game Theory Evaluation}
```
Comment (jserv) [inline paper.tex:188]: It is the first time where Game Theory takes place. Before that, you shall mention in introduction and even related work sections.

File: paper.tex [hunk 3]
```diff
@@ -136,6 +136,34 @@ \subsection{Refunding}
...
+\item  More than 51\% of data buyers are amoung complete information exchange state. In other words, once one of them launches voting for stopping buying data and refunding, they would succeed in refund voting.
+\item  One of the data buyers who has complete imformation exchange with other 51\% buyers will definitely find out the wrong or fake data provided by data provider is wrong.
+\end{itemize}
+
+Based on the description above, the discount sum of payoff for our repeated game is 
+\begin{equation} \label{payoff}
+\sum_{i=0}^{m}{\beta^{in}\cdot [(\sum_{j=0}^{n} \beta^j \cdot p_{fee}) - C_{maintain}]}
+\end{equation}
+
+According to Ashraf E. Al-Fagih's work\cite{DataPrice} we learned $p_{fee}$ is a sigmoid function of $\pi_{correct}$, and based on rule of thumb $C_{maintain}$ is about a exponential function of $\pi_{correct}$. We can substitute $p_{fee}$ and $C_{maintain}$ with $\pi_{correct}$ into Eq. (\ref{payoff}), then we can find out the Nash Equilibrium of repeated game with differential easily.
```
Comment (jserv) [inline paper.tex]: Change "According to Ashraf E. Al-Fagih's work" to "Al-Fagih et al. evaluated ..."

Reference: https://wordvice.com.tw/%E6%96%87%E7%8D%BB%E5%BC%95%E7%94%A8%E6%A0%BC%E5%BC%8F%E8%88%87et-al-%E7%9A%84%E4%BD%BF%E7%94%A8%E6%96%B9%E6%B3%95/

File: paper.tex [hunk 4]
```diff
@@ -18,7 +18,7 @@
 %
 \author{
 %%% Fill in name here
-\IEEEauthorblockN{Zan-Jun Wang\IEEEauthorrefmark{3}, Ching-Hua Lin\IEEEauthorrefmark{2}, Ching-Chun (Jim) Huang\IEEEauthorrefmark{1}}\\
+\IEEEauthorblockN{Yang-Hao Yuan\IEEEauthorrefmark{4}}, Zan-Jun Wang\IEEEauthorrefmark{3}, Ching-Hua Lin\IEEEauthorrefmark{2}, Ching-Chun (Jim) Huang\IEEEauthorrefmark{1}}\\
```
Comment (jserv) [inline paper.tex]: Shift your name right after Ching-Hua.

File: paper.tex [hunk 5]
```diff
@@ -136,6 +136,34 @@ \subsection{Refunding}
...
 
+\subsection{Game Theory Evaluation}
+
+\subsubsection{Game Theory Scenario}
+In our model there are two kinds of participants. One is data provider, and the other one is data buyer. Data provider would receive subscription fee for buying data each time the data is transmitted to data buyer.
+The data buyer will keep buying data from data povider, only when the data sold by the data provider with high enough correct rate, $\pi_{correct}$. We deonte this correct rate threshold with $\pi_{threshold}$.
+Once the correct rate is lower than $\pi_{threshold}$, the data buyers would consider this data provider as a malicious data provider, and reject to buy the data from this data provider.
+
+\subsubsection{Model Description}
+We use a repeated game to express the scenario of our Data Marketplace. Figure.\ref{fig:decision_tree} shows the decision tree to depict the the repeated game we used. Each level in this decision tree represents each round of data transmission from data provider to data buyers, and the data buyers would pay $p_{fee}$ in total to buy the data.
```
Comment (jserv) [inline paper.tex]: Do we have to explain how "repeated game" works for Game Theory?

File: paper.tex [hunk 6]
```diff
@@ -18,7 +18,7 @@
 %
 \author{
 %%% Fill in name here
-\IEEEauthorblockN{Zan-Jun Wang\IEEEauthorrefmark{3}, Ching-Hua Lin\IEEEauthorrefmark{2}, Ching-Chun (Jim) Huang\IEEEauthorrefmark{1}}\\
+\IEEEauthorblockN{Zan-Jun Wang\IEEEauthorrefmark{3}, Ching-Hua Lin\IEEEauthorrefmark{2}, Yang-Hao Yuan\IEEEauthorrefmark{4}}, Ching-Chun (Jim) Huang\IEEEauthorrefmark{1}}\\
```
Comment (jserv) [inline paper.tex]: Fix format.

File: paper.tex (hunk 5 above)
Comment (jserv) [reply paper.tex]: ?

Comment (howjmay) [reply paper.tex]: So I should present some simple instroction of repeated game here ?

Comment (jserv) [reply paper.tex]: For formal writing, you shall explain in  advance.

File: paper.tex [hunk 7]
```diff
@@ -136,6 +138,34 @@ \subsection{Refunding}
...
+
+\begin{figure} \centering \includegraphics[width=3.3in]{img/decision_tree.png} \caption{Decision Tree}
+    \label{fig:decision_tree} \end{figure}
+For every $n$ round, the data provider would cost $C_{maintain}$ to maintain or enhance the sensors that data provider owns. We call this as a maintain period. Moreover, the value of data and the cost of maintenance will decrease as time pass, so we introduce a discount factor $\beta$ to depict this phenomena.
+
+We have these following assumptions:
+\begin{itemize}
+\item  There is no any other data providers provide the same product in Data Marketplace. Therefore, we can consider each provider's behavior independently. That means this decision tree represent the behavior of only one data provider.
+\item  More than 51\% of data buyers are amoung complete information exchange state. In other words, once one of them launches voting for stopping buying data and refunding, they would succeed in refund voting.
+\item  One of the data buyers who has complete imformation exchange with other 51\% buyers will definitely find out the wrong or fake data provided by data provider is wrong.
```
Comment (jserv) [inline paper.tex]: Explicitly use proper terms rather than "wrong/fake".

**Pull request merged**